requests-for-comments/2024-06-21-warehouse-launch-plan.md

```diff
@@ -0,0 +1,51 @@
+## [Marketing] Data warehouse launch plan
+
+The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 
+
+### Ahead of launch
+
+- [ ] Publish a Stripe data warehouse tutorial - Marketing
+- [ ] Publish a Hubspot data warehouse tutorial - Marketing
+- [ ] Publish a Postgres data warehouse tutorial - Marketing
+- [ ] Update docs as needed - Marketing
+- [ ] Add Data Warehouse to in-app 'Explore the docs' menu - @joethreepwood
+- [x] Key art - @lottiecoxon (we have some already, open to new though)
+
```

Comment:
Some more ideas for tutorials here:

- Joining your [stripe/hubspot/zendesk/etc] data with PostHog for filtering within the product
- Using your postgres app data in PostHog

Reply:
Great. I'm going to defer to @andyvan-ph @ivanagas and @Lior539 on how we tackle these tutorials and when. Hopefully we can get a few of these out ASAP.

Reply:
From the sprint chat, sounds like we should just focus on the tutorial + template stuff for now and not worry too much about promotional activity just yet.

@Lior539 @ivanagas Shall we nail down a list of tutorials we want to focus on first, and divide them up?

Would be good to have your input here @EDsCODE, but is it more useful to have:

1. Very straightforward guides like "How to integrate and query your Stripe data"
2. Or more problem orientated ones like "How to analyze Google Ad ROI in PostHog"

Initially, I was thinking the latter, but maybe it's better to start with very simple setup guides, then follow-up for more specific "use case" ones?

In any case, I think we could end doing a lot of work here as users will need a lot more hand holding to learn how to use warehouse / why it could be useful for them.

Reply:
Lets do:
- How to set up Stripe reports
- How to set up Zendesk reports
- How to set up Hubspot reports

Each of these have solid search and relatively light competition. We can use them to:
1. Build a related dashboards which then we can convert to a template
2. Do an example in each of joining and combining source and product data

Reply:
I'll write issues for all these today.

Reply:
- Stripe: https://github.com/PostHog/posthog.com/issues/8662
- Zendesk: https://github.com/PostHog/posthog.com/issues/8663
- Hubspot: https://github.com/PostHog/posthog.com/issues/8664

Reply:
I think we should do a Google Ad ROI at some point as well. Eric mentioned it as something people are doing already. Also, bonus point, we can set it up for ourselves. 😅

Reply:
We should definitely prioritize tutorials that are useful in multiple ways. The ones I mention are useful as tutorials, SEO pieces, and help build dashboards. Google Ad ROI will be similar :)